components/Form/ApplicationForm/components.tsx

```diff
@@ -103,12 +104,12 @@ export const PersonalInfo = ({ session, register, errors }: Props) => (
 
 export const Education = ({ register, errors }: Props) => (
   <Group title="Education">
-    <Input
-      type="text"
+    <Select
```

Comment:
looks gud jus a slight styling problem

rn it looks like dis 
![image](https://user-images.githubusercontent.com/62922309/215033757-a03fb8d1-ad8d-4d54-9502-11bcdaced645.png)

but the options shud be contained within the box; like dis
![image](https://user-images.githubusercontent.com/62922309/215033858-dc3f76dc-6bfe-47a4-bac2-41ebd9540602.png)